package.json

```diff
@@ -117,6 +117,7 @@
     "*.{js,cjs,mjs}": "eslint --max-warnings=0",
     "*.ts": "eslint --max-warnings=0 --plugin tsc --rule 'tsc/config: [2, {configFile: \"./tsconfig.json\"}]'",
     "*.md": "markdownlint",
+    "*.json": "dprint fmt",
```

Comment:
Should eslint also handle json via dprint? Having it here will handle it via husky, but nothing will catch it later.

Reply:
not sure if eslint recognize json file

Reply:
I think there are some plugins?

Alternatively, maybe we can remove this line until eslint is configured to handle it? Otherwise, this is autoformatting at the lint-staged step (which can be skipped) and CI won't catch json.

Reply:
Don't think it's necessary, Jason is a pretty simple language. I add this just because dprint has the ability to format json.

Reply:
It's both not changing commit in hooks, but also the CI lint step not throwing errors if json isn't formatted.